libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/logger/SalesforceLogger.java

```diff
@@ -148,8 +212,8 @@ private boolean isDebugMode() {
                     }
                 }
             }
-        } catch (PackageManager.NameNotFoundException e) {
-            debugMode = true;
+        } catch (NameNotFoundException e) {
+            /* Intentionally blank */
```

Comment:
This seems like a really strange assumption we had here.  Without looking too deeply, good change 👍